platform/binance/client.go

```diff
@@ -16,10 +16,15 @@ type Client struct {
 	blockatlas.Request
 }
 
+const (
+	TokensLimit       = "1000"
+	TransactionsLimit = "25"
```

Comment:
`"25"` => `25`

make this a global parameter, as we always return the last 25

Reply:
https://github.com/trustwallet/blockatlas/pull/1010/commits/7559cf323139c706b8e44cfb911e7eed2f5106d6